src/controllers/pages/index.js

```diff
@@ -24,4 +24,11 @@ export default class IndexController {
             hashHistory.push('/overdue');
         });
     }
+
+    static getStudents(){
```

Comment:
Is this the thing i'm not supposed to see? Mind giving some info on what this does before I approve? Seems to be a mix of students and models for some reason.